webextensions/manifest/optional_permissions.json

```
@@ -272,6 +272,67 @@
             }
           }
         },
+        "declarativeNetRequest": {
+          "__compat": {
+            "description": "<code>declarativeNetRequest</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "declarativeNetRequestFeedback": {
+          "__compat": {
+            "description": "<code>declarativeNetRequestFeedback</code>",
+            "support": {
+              "chrome": {
+                "version_added": "84",
+                "notes": "Available only to unpacked extensions."
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "113"
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15.4"
+              },
+              "safari_ios": "mirror"
+            }
+          }
+        },
+        "declarativeNetRequestWithHostAccess": {
+          "__compat": {
+            "description": "<code>declarativeNetRequestWithHostAccess</code>",
+            "support": {
+              "chrome": {
+                "version_added": "84"
```

```suggestion
                "version_added": "96"
```

Permission was introduced in 95 -  https://chromium.googlesource.com/chromium/src.git/+/2bfaa6f0446e2fc82485c53e351e1e835a0a2edc for the commit, and VERSION of that commit: https://chromium.googlesource.com/chromium/src.git/+/2bfaa6f0446e2fc82485c53e351e1e835a0a2edc/chrome/VERSION

And finally enabled for (stable) release in 96 through https://chromium.googlesource.com/chromium/src.git/+/a3d03e2823ab6c431e8e56ac10d5abac12881e09 - with the following VERSION: https://chromium.googlesource.com/chromium/src.git/+/a3d03e2823ab6c431e8e56ac10d5abac12881e09/chrome/VERSION